Libraries/LibWeb/Painting/DisplayListPlayerSkia.cpp

```diff
@@ -204,12 +224,11 @@ void DisplayListPlayerSkia::push_stacking_context(PushStackingContext const& com
 {
     auto& canvas = surface().canvas();
 
-    auto affine_transform = Gfx::extract_2d_affine_transform(command.transform.matrix);
-    auto new_transform = Gfx::AffineTransform {}
-                             .translate(command.transform.origin)
-                             .multiply(affine_transform)
-                             .translate(-command.transform.origin);
-    auto matrix = to_skia_matrix(new_transform);
+    auto new_transform = Gfx::FloatMatrix4x4::identity();
+    new_transform = new_transform * Gfx::translation_matrix(Vector3<float>(command.transform.origin.x(), command.transform.origin.y(), 0));
```

Comment:
You can start off with `Gfx::translation_matrix` instead of first assigning the identity matrix right?

Reply:
Good point, done that now.